packages/melos/lib/src/common/git_repository.dart

```diff
@@ -89,14 +90,19 @@ class GitHubRepository extends HostedGitRepository with SupportsManualRelease {
     throw FormatException('The URL $uri is not a valid GitHub repository URL.');
   }
 
+  static const kDefaultBase = 'https://github.com';
```

Comment:
https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters

Reply:
Seems like I've gotten way to used to this from all the UI constants in Flutter...

Reply:
Yeah, they sadly don't follow their own recommendations, but I think it's mostly to avoid breaking changes.